### PR TITLE
add: `exercism-cli-bin`

### PR DESCRIPTION
**Pull request by raindev**
Tested both on an x86_64 laptop and on an amd64 Pi 4.

### CODE REVIEW

File: packages/exercism-cli/exercism-cli.pacscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+arch=('amd64' 'arm64')
+if [[ ${CARCH} == amd64 ]]; then
+    hash="4ea3e1ea8916a8003da95dbd6eef7a3a29802e637ed6a0f2aaaa2f1c98754915"
+    arch_suffix="x86_64"
+else
+    hash="57f9620e5346bf045ca72b581cb106f9dc56841ac1ff0a3070188c49d5d0b573"
+    arch_suffix="arm64"
+fi
+url="https://github.com/exercism/cli/releases/download/v${pkgver}/exercism-${pkgver}-linux-${arch_suffix}.tar.gz"
+pkgdesc=" A Go based command line tool for exercism.org"
```
Comment (Elsie19) [inline packages/exercism-cli/exercism-cli.pacscript:13]:
```suggestion
pkgdesc="Go based command line tool for exercism.org"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Also this should be `exercism-cli-bin`

**Comment by raindev** [comment]
I wasn't sure whether to call the package exercism-cli (more precise) or just exercism (more concise).

**Comment by Elsie19** [comment]
well you are installing a binary, so it should be either `exercism-bin` or `exercism-cli-bin` if there is a gui alternative.

**Comment by raindev** [comment]
@Elsie19 by the same extent the watchexec package should also have been named with -bin suffix, I take?

**Comment by Elsie19** [comment]
> @Elsie19 by the same extent the watchexec package should also have been named with -bin suffix, I take?

👀 actually yeah can you make a PR to fix that?

**Comment by raindev** [comment]
> well you are installing a binary, so it should be either exercism-bin or exercism-cli-bin if there is a gui alternative.

There's no GUI ([used to be](https://github.com/exercism/gui)) alternative but the exercism website itself is also open source.

**Comment by raindev** [comment]
@Elsie19 I'm sorry for missing this in the previous PR, fixed in https://github.com/pacstall/pacstall-programs/pull/5133

**Comment by Elsie19** [comment]
> @Elsie19 I'm sorry for missing this in the previous PR, fixed in https://github.com/pacstall/pacstall-programs/pull/5133

it's ok it was my fault 😳

File: packagelist [hunk 2]
```diff
@@ -101,6 +101,7 @@ enpass-deb
 epson-inkjet-printer-stylus-nx110-series-deb
 epson-printer-utility-deb
 exa-git
+exercism
```
Comment (Elsie19) [inline packagelist:104]:
```suggestion
exercism-cli-bin
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

File: packagelist [hunk 3]
```diff
@@ -101,7 +101,7 @@ enpass-deb
 epson-inkjet-printer-stylus-nx110-series-deb
 epson-printer-utility-deb
 exa-git
-exercism
+exercism-cli-bin
```
Comment (raindev) [inline packagelist:104]: That was sloppy of me, thanks for fixing!

Comment (raindev) [reply packagelist:104]: It's quite likely I'll keep forgetting packagelist so I added a commit hook that automates this in https://github.com/pacstall/pacstall-programs/pull/5139

Comment (Elsie19) [reply packagelist:104]: Thanks!

**Pull request merged**